### DISCUSSION TITLE
Monitor mountpoint (instead of disks)

### DISCUSSION THREAD

**Discussion by FuzzyMistborn** [bkbilly/lnxlink#105] [Q&A] [answered]
I run mergerfs on my main server and would like to track how full it is.  From my experimentation LNXlink can monitor disks, but I can't figure out how to monitor mount points.  Am I overlooking something?

**Comment by bkbilly** [comment]
I am not familiar with the mergerfs, but I could create a new module for the mounts.

**Reply by FuzzyMistborn** [reply]
https://github.com/trapexit/mergerfs

It's a union file system, so I take a bunch of disks and they get merged into one mount point.  Think ZFS, but with whatever drives you have lying around.

**Reply by bkbilly** [reply]
It seems very useful!
The only way I found to get the mount size is the df. Does it work on your mounts?

**Reply by FuzzyMistborn** [reply]
Yes, df works

**Comment by bkbilly** [comment, marked as answer]
I've pushed an update with the new module called mounts.
You can manually configure it to add the mounts that you want to check.

I've also added the ability to automatically find the remote mounts on the system and expose their status.
I am not sure if it works on other distros other than Gnome, but for Ubuntu it works as long as it is mounted with GVFS which is the default for the file manager.

**Reply by FuzzyMistborn** [reply]
Nice!  Wasn't able to install as a custom module, but looking forward to this when you release it!

**Reply by bkbilly** [reply]
I haven't made any changes on the core application for the mounts module, so you should be able to install it using custom module.
In a few days I'll have the release ready, so if you are not in a hurry, you will find it there.
A new configuration is going to be added on your configuration file that looks like this:
```yaml
settings:
  mounts:
    autocheck: false
    directories: []
```

**Reply by FuzzyMistborn** [reply]
There's a new requirement (pyobject), wasn't sure how to handle.

**Reply by bkbilly** [reply]
The requirements are installed automatically on each module, so you don't have to worry about it.

**Reply by FuzzyMistborn** [reply]
Didn't work for me:

```
ERROR:lnxlink:Error with module mount: attempted relative import with no known parent package
ERROR:lnxlink:Addon mount is not supported, please remove it from your config: No module named 'lnxlink.modules.mount'
```

I'll just wait for the release.

**Reply by bkbilly** [reply]
Oh, it seems that there is an issue with the custom modules.
I'll look into it, thanks for letting me know!

**Reply by bkbilly** [reply]
I've made some changes to the mounts module and now it should be working.

**Reply by FuzzyMistborn** [reply]
Happy to report the latest release worked just as expected. Thank you so much!
